src/gasman.c

```diff
@@ -324,7 +324,7 @@ void CHANGED_BAG_IMPL(Bag bag) {
 #define SpaceBetweenPointers(a,b) (((UInt)((UInt)(a) - (UInt)(b)))/sizeof(Bag))
 
 #define SizeMptrsArea SpaceBetweenPointers(OldBags, MptrBags)
-#define SizeOldBagsArea SpaceBetweenPointers(YoungBags,OldBags)
+#define SizeOldBagsArea SpaceBetweenPointers(YoungBags, OldBags)
 #define SizeYoungBagsArea SpaceBetweenPointers(AllocBags, YoungBags)
 #define SizeAllocationArea SpaceBetweenPointers(StopBags, AllocBags)
 #define SizeUnavailableArea SpaceBetweenPointers(EndBags, StopBags)
```

Comment:
OK, what I missed was in particular the separate allocation area, and also the big comment around line 266.

But I still don't understand what this "unavailable" area is needed for...